snafu/cyclictest_wrapper/cyclictest_wrapper.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import os
+import argparse
+
+from .trigger_cyclictest import Trigger_cyclictest
+
+
+class cyclictest_wrapper():
+
+    def __init__(self, parent_parser):
+        parser_object = argparse.ArgumentParser(description="Cyclic Test Wrapper script",
+                                                parents=[parent_parser],
+                                                formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+        parser = parser_object.add_argument_group("Cyclictest benchmark")
+        parser.add_argument(
+            '-p', '--path',
+            required=True,
+            help='Path to cyclictest script')
+        parser.add_argument(
+            '-s', '--samples', type=int, default=1,
+            help='Number of times to run the benchmark')
+        parser.add_argument(
+            '-u', '--uuid',
+            required=True,
+            help='Provide the uuid')
+        parser.add_argument(
+            '--user',
+            required=True,
```

Comment:
Isn't required when providing a default a bit contradictory? I.e. if theres a default so I shouldn't have to provide a value

Reply:
Fixed